### DISCUSSION TITLE
Include in "where", an "OR" clause containing columns of connected tables

### DISCUSSION THREAD

**Discussion by AndreinaRiera** [prisma/prisma#16928] [Q&A] [answered]
Hello! 
I was looking in the documentation, but I didn't find any section where this is discussed. 

I would like to do something like:

const users = await this.prisma.user.findMany({
        include: {
          createdBy: true,
        },
        where: {
          OR: [{ uuid: userAuth.uid }, { createdBy: { uid: userAuth.uid } }],
        },
      });

In this case, in a "findMany" query, I want to fetch all users that, their "uuid" is equal to the one specified, or else, that the "uuid" in the connected table by "createdBy" is equal to the specified value. 

Could you please guide me on how to do this? please

( by the way... It would be amazing if there was a SQL query  to prisma code converter )

**Comment by nurul3101** [comment, marked as answer]
Hello @AndreinaRiera 👋 
Can you please share the `User` model? 
Are you getting any errors while invoking this query?

**Reply by AndreinaRiera** [reply]
Hello @nurul3101!
Sorry, it was my mistake 😅. 
I analyzed the model of that table again and saw where I was going wrong. I was missing one more nesting step. 
Anyway, thank you very much for your disposition and for answering my question!

**Reply by nurul3101** [reply]
No worries at all, I am glad to hear that the issue is resolved now.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
